ghost/core/core/server/api/endpoints/utils/serializers/output/mappers/emails.js

```diff
@@ -13,5 +14,11 @@ module.exports = (model, frame) => {
         );
     });
 
+    if (!labs.isSet('emailErrors')) {
+        if (jsonModel.status === 'failed') {
+            jsonModel.status = 'submitted';
```

Comment:
This would mean we'll show `submitted` for a completely failed email as well right ? 🤔